src/states/node.rs

```diff
@@ -722,43 +728,79 @@ impl Node {
     // Acknowledge reception of the message and broadcast to our section if necessary
     // The function is only called when we are in the destination authority
     fn ack_and_broadcast(&mut self,
-                         signed_msg: &SignedMessage,
+                         signed_msg: SignedMessage,
                          route: u8,
                          hop_name: XorName,
                          sent_to: &BTreeSet<XorName>) {
         self.send_ack(signed_msg.routing_message(), route);
         // If the destination is our section we need to forward it to the rest of the section
         if signed_msg.routing_message().dst.is_multiple() {
             if let Err(error) = self.send_signed_message(signed_msg, route, &hop_name, sent_to) {
-                debug!("{:?} Failed to send {:?}: {:?}", self, signed_msg, error);
+                debug!("{:?} Failed to send signed message: {:?}", self, error);
```

Comment:
Why was the actual message removed from the debug output here?  Same question for line 817 below.